.gitmodules

```diff
@@ -72,3 +72,9 @@
 [submodule "src/jaegertracing/opentelemetry-cpp"]
 	path = src/jaegertracing/opentelemetry-cpp
 	url = https://github.com/open-telemetry/opentelemetry-cpp.git
+[submodule "src/lmdb-safe"]
+	path = src/lmdb-safe
+	url = https://github.com/ahupowerdns/lmdb-safe.git
+[submodule "src/rgw/driver/posix/lmdb-safe"]
```

Comment:
Do we need to include the submodule twice?

Reply:
is it?  will fix